install.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Clone simple-taiko-node
+# git clone https://github.com/taikoxyz/simple-taiko-node.git
+# cd simple-taiko-node
+# printf "\n"
+
+echo Taiko Node Setup Script v0
+
+# Copy the sample .env files
+echo Getting environmental variables...
+cp .env.sample .env
+
+# get api key
+echo Setting L1 archive node endpoints
+read -p 'Enter HTTP endpoint: ' http_key
+read -p 'Enter WS endpoint: ' ws_key
+# set L1_ENDPOINT_HTTP and  L1_ENDPOINT_WS
+sed  '/\[L1_ENDPOINT_HTTP\]/a http_key' .env > /dev/null
+sed  '/\[L1_ENDPOINT_WS\]/a ws_key' .env > /dev/null
+printf "\xE2\x9C\x94 Configuration was successful! \n"
+printf "\n"
+
+# If you ran a testnet node previously, remove old volumes
+#docker compose down -v
+
+# Start a node using Docker
+echo Starting docker container...
+sudo docker compose up -d > /dev/null
+printf "\xE2\x9C\x94 Node started! \n"
+printf "\n"
+
+# Verify node is running
+echo Verifying node is running...
+
+# Check if the Execution Layer client is connected to Taiko L2
+echo Check the Execution Layer client is connected to Taiko L2
+curl -s http://localhost:8547 \
```

Comment:
i would recommend reading these port values from the .env.sample file in case they change. same with line 68